scipy/stats/_stats_py.py

```diff
@@ -4461,7 +4473,7 @@ def binary_search(n, n1, n2, side):
         pexact = hypergeom.pmf(c[0, 0], n1 + n2, n1, n)
         pmode = hypergeom.pmf(mode, n1 + n2, n1, n)
 
-        epsilon = 1 - 1e-4
+        epsilon = 1 - 1e-14
         if np.abs(pexact - pmode) / np.maximum(pexact, pmode) <= 1 - epsilon:
```

Comment:
I would prefer to make `epsilon` be `1e-14` and use `1 - epsilon` elsewhere, but I didn't want to confuse this PR with a refactoring.